[IMP] Productivity: Discuss Getting Started

tiku-odoo opened the pull request:
Task: https://www.odoo.com/web#id=3113715&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form


Update Getting Started Discuss page

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/4645).

hojo-odoo commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,56 +1,61 @@
 
-The messages tagged as *Mark as Todo* are also shown in *Starred*, while the ones *Marked as Read*
-are moved to *History*.
+The messages tagged as :guilabel:`Mark as Todo` are also shown in :guilabel:`Starred`, while the
+ones :guilabel:`Marked as Read` are moved to :guilabel:`History`.
```
```suggestion
Clicking :guilabel:`Mark as Todo` on a message causes it to appear 
on the :guilabel:`Starred` page, while clicking :guilabel:`Mark as Read` moves the message to :guilabel:`History`.
```

hojo-odoo commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,56 +1,61 @@
 
-To start a chat, click on *New Message* on the *Messaging Menu*, or go to *Discuss* and send a
-*Direct Message*.
+To start a chat go to *Discuss* app and click on the :guilabel:`+` icon next to :guilabel:`Direct
```
We're telling the user to select the Discuss app, so it should have a menuselection tag.
```suggestion
To start a chat, go to the :menuselection:`Discuss` app and click on the :guilabel:`+` icon next to :guilabel:`Direct
```

hojo-odoo commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -59,13 +64,12 @@ To start a chat, click on *New Message* on the *Messaging Menu*, or go to *Discu
-  *#channel-name*.
-| A notification is sent to the user mentioned either to his *Inbox* or through email, depending
-  on his settings.
+To mention a user within a chat or the chatter type *@user-name*; to refer to a channel, type
```
Add comma
```suggestion
To mention a user within a chat or the chatter, type *@user-name*; to refer to a channel, type
```

hojo-odoo commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -59,13 +64,12 @@ To start a chat, click on *New Message* on the *Messaging Menu*, or go to *Discu
-| A notification is sent to the user mentioned either to his *Inbox* or through email, depending
-  on his settings.
+To mention a user within a chat or the chatter type *@user-name*; to refer to a channel, type
+*#channel-name*. A notification is sent to the user mentioned either to the *Inbox* or through an
+email, depending on the settings.
```
Rewrite
```suggestion
*#channel-name*. The user mentioned will be notified in their *Inbox* or through an email, depending on their communication settings.
```

hojo-odoo commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -76,11 +80,11 @@ Mentions in the chat and on the Chatter
 
 It is helpful to see what colleagues are up to and how quickly they can respond to messages by
-checking their *Status*. The status is shown on the left side of a contact’s names on Discuss’
+checking their *status*. The status is shown on the left side of a contact’s names on Discuss’
```
```suggestion
checking their *status*. The status is shown on the left side of a contact’s name on the Discuss
```

hojo-odoo reviewed (commented):
Hey @tiku-odoo,

I just had a few comments on this. In terms of things that need to be updated throughout the doc, make sure to italicize the Discuss app any time it's not inside of a menuselection tag.

Thanks,
hojo

hojo-odoo reviewed (commented):
Hey @tiku-odoo,

I just had a few comments on this. In terms of things that need to be updated throughout the doc, make sure to italicize the Discuss app any time it's not inside of a menuselection tag.

Thanks,
hojo

tiku-odoo commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,56 +1,61 @@
 
-.. image:: get_started/inbox_actions.png
+By choosing :guilabel:`Handle in Odoo`, the above notifications are shown in the *Dicuss* app's
+:guilabel:`Inbox`. Messages can then be :guilabel:`Marked as Todo`, :guilabel:`Replied`,
+:guilabel:`Marked as Read` or responded with an :guilabel:`emoji`.
```
Noted; I've changed all gui labels to represent the label on screen.

tiku-odoo commented:
@samueljlieber This document is ready for 1st round of technical review. Thanks for your help on this. 

:+1:

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,71 +1,78 @@
 
-To start a chat, click on *New Message* on the *Messaging Menu*, or go to *Discuss* and send a
-*Direct Message*.
+To start a chat, go to the :menuselection:`Discuss` app and click on the :guilabel:`+` icon next to
+:guilabel:`Direct Messages` or :guilabel:`Channels` in the left menu of the dashboard.
```
Icon formatting– add the identifier in parenthesis please :)
```suggestion
To start a chat, go to the :menuselection:`Discuss` app and click on the :guilabel:`+ (plus)` icon
next to :guilabel:`Direct Messages` or :guilabel:`Channels` in the left menu of the dashboard.
```

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,71 +1,78 @@
+send desktop notifications for chats. If accepted, the user will receive push notifications on their
+desktop for the messages they receive, regardless of where the user is in Odoo.
 
 .. image:: get_started/odoobot_push.png
```
This image is too wide at 944px, can you please resize it to be 768px wide (933px max), also the image filename contains an underscore, please change it to hyphens

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,71 +1,78 @@
+To start a chat, go to the :menuselection:`Discuss` app and click on the :guilabel:`+` icon next to
+:guilabel:`Direct Messages` or :guilabel:`Channels` in the left menu of the dashboard.
 
 .. image:: get_started/channels_direct_messages.png
```
underscores in filename here too, please update to hyphens :)

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,71 +1,78 @@
+   :alt: View of Discuss’s panel emphasizing the titles channels and direct messages in Odoo Discuss
 
-You can also create :doc:`public and private channels <team_communication>`.
+A company can also easily create :doc:`public and private channels <team_communication>`.
```
please update to use global relative path:
```suggestion
A company can also easily create :doc:`public and private channels
</applications/productivity/discuss/overview/team_communication>`.
```

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,71 +1,78 @@
-  *#channel-name*.
-| A notification is sent to the user mentioned either to his *Inbox* or through email, depending
-  on his settings.
+To mention a user within a chat or the chatter, type *@user-name*; to refer to a channel, type
+*#channel-name*. The user mentioned will be notified in their *Inbox* or through an email, depending
+on their communication settings.
```
User input should use literal formatting not italics, and _inbox_ should be lowercase since the text is not in a guilabel
```suggestion
To mention a user within a chat or the chatter, type `@user-name`; to refer to a channel, type
`#channel-name`. The user mentioned will be notified in their *inbox* or through an email, depending
on their communication settings.
```

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,71 +1,78 @@
+*#channel-name*. The user mentioned will be notified in their *Inbox* or through an email, depending
+on their communication settings.
 
 .. image:: get_started/chat_windows.png
```
Image filename- update to use hyphens please :) and add a period to the alt tag

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -76,12 +83,12 @@ Mentions in the chat and on the Chatter
    task’s followers, and secondly on *Employees*. If the record being searched does not match with
    either a follower or employee, the scope of the search becomes all partners.
```
Apostrophe fix and no need for italics on employees
```suggestion
   task's followers, and secondly on employees. If the record being searched does not match with
   either a follower or employee, the scope of the search becomes all partners.
```

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -76,12 +83,12 @@ Mentions in the chat and on the Chatter
 It is helpful to see what colleagues are up to and how quickly they can respond to messages by
-checking their *Status*. The status is shown on the left side of a contact’s names on Discuss’
-sidebar and on the *Messaging menu*.
+checking their *status*. The status is shown on the left side of a contact’s name on the *Discuss*
+sidebar, on the *Messaging menu* and when listed in the *chatter*.
```
Apostrophe fix, can use guilabel on Discuss since it is displayed on the UI in this instruction, lowercase messaging menu
```suggestion
checking their *status*. The status is shown on the left side of a contact's name on the
:guilabel:`Discuss` sidebar, on the *messaging menu* and when listed in the *chatter*.
```

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -95,4 +102,4 @@ sidebar and on the *Messaging menu*.
 
 .. seealso::
    - :doc:`team_communication`
-   - :doc:`/applications/general/email_communication/email_servers`
+   - :doc:`plan_activities`
```
```
   - :doc:`/applications/productivity/discuss/overview/team_communication`
   - :doc:`/applications/productivity/discuss/overview/plan_activities`
```

samueljlieber reviewed (changes requested):
Hi @tiku-odoo! Nice job with the improvements to this doc! I have a few technical corrections and some suggestions for improvement. Please see below and let me know if you have any questions!

Regarding the curly apostrophes, there is a difference between this `’` and this `'` apostrophe. Google Docs seems to automatically switch them, so they need to be converted back to normal apostrophes (`'`). OR follow these instructions for disabling smart quotes in Google Docs to avoid curly apostrophes:
1. Open your Google Docs document.
2. Click on "File" in the menu at the top left corner.
3. Select "Preferences" from the dropdown menu. A pop-up window will appear.
4. In the pop-up window, locate the "Smart quotes" option. It is usually under the "General" tab.
5. Uncheck the box next to "Smart quotes" or any similar option that refers to curly quotes or smart typography.
6. Click "OK" to save the changes and close the Preferences window.
7. Now, you can copy the desired text from Google Docs and paste it into Visual Studio Code without the curly apostrophes being introduced.

tiku-odoo commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,71 +1,78 @@
-  you were mentioned or that you follow, to be sent through email. By choosing *Handle in Odoo*,
-  they are shown in the *Inbox*.
-| Messages can then be *Marked as Todo*, *Replied*, or *Marked as Read*.
+By default, the field is set as :guilabel:`Handle by Emails`. With this setting enabled, a
+notification email will be sent by Odoo every time a message is sent, a note is sent with an '@'
+mention, or a notification is sent for a record that the user follows.
```
Nice catch, thanks for suggesting!

tiku-odoo commented:
@samueljlieber This document is ready for a last look when you have a moment. I've made all the requested changes and edits to the images/file names. :+1: 

Thanks for your help on these!

samueljlieber commented on content/applications/productivity/discuss/overview/team_communication.rst:
```diff
@@ -1,16 +1,16 @@
 
-A *Public* channel can be seen by everyone, while a *Private* one is only visible to users invited
-to it.
+A *pulic* channel can be seen by everyone, while a *private* one is only visible to users invited to
+it.
```
Typo
```suggestion
A *public* channel can be seen by everyone, while a *private* one is only visible to users invited
to it.
```

samueljlieber commented on content/applications/productivity/discuss/overview/team_communication.rst:
```diff
@@ -1,16 +1,16 @@
+A *pulic* channel can be seen by everyone, while a *private* one is only visible to users invited to
+it.
 
 .. image:: team_communication/create_channel.png
```
Update filename please
```suggestion
.. image:: team_communication/create-channel.png
```

Also please update the alt tag for this image:

>    :alt: View of the Discuss sidebar and a channel being created in Odoo Discuss.

samueljlieber commented on content/applications/productivity/discuss/overview/team_communication.rst:
```diff
@@ -25,21 +25,22 @@ to it.
 Configuration options
```
Line 22-23 Early line break (I cant make a suggestion there)

```
   announcements), whereas a private channel could be used whenever information should be limited to
   specific groups (such as a specific department).
```

samueljlieber commented on content/applications/productivity/discuss/overview/team_communication.rst:
```diff
@@ -25,21 +25,22 @@ to it.
 
-You can configure a channel’s name, description, email alias, and privacy by clicking on the
-*Channel Settings* icon on the sidebar.
+A channel’s name, description, email alias, and privacy can be configured by clicking on the
+*channel settings* ' ⚙️ ' icon on the sidebar.
```
Icon formatting
```suggestion
A channel's name, description, email alias, and privacy can be configured by clicking on the channel
settings :guilabel:`⚙️ (gear)` icon on the sidebar.
```

samueljlieber commented on content/applications/productivity/discuss/overview/team_communication.rst:
```diff
@@ -25,21 +25,22 @@ to it.
    :align: center
    :alt: View of a channel’s settings form in Odoo Discuss
 
 Privacy and Members
```
I _think_ members should be lowercase here
```suggestion
Privacy and members
```

samueljlieber reviewed (changes requested):
Hi @tiku-odoo 👋 Thanks for updating `get_started.rst` 🙂 looking good except for a small change– please remove the unused image `handle-by-emails.png` from `get_started/`.

I missed `team_communication.rst` on my first review, please see my technical suggestions below, ultimately if we're already improving a doc we may as well correct the rst. I was unable to add suggestions to the rest of the doc after line 44, so please continue to make the corrections regarding italics, guilabels, image file names, periods on alt tags, and menuselection use.

Thank you for your continued hard work here 🙏

tiku-odoo commented:
> Hi @tiku-odoo wave Thanks for updating `get_started.rst` slightly_smiling_face looking good except for a small change– please remove the unused image `handle-by-emails.png` from `get_started/`.
> 
> I missed `team_communication.rst` on my first review, please see my technical suggestions below, ultimately if we're already improving a doc we may as well correct all of the rst. I was unable to add suggestions to the rest of the doc after line 44, so please continue to make the corrections regarding italics, guilabels, image file names, periods on alt tags, and menuselection use.
> 
> Also please perform a [search and replace](https://learn.microsoft.com/en-us/visualstudio/ide/finding-and-replacing-text?view=vs-2022) on this doc to convert the curly apostrophes to straight. In VSCode press `CTRL + SHIFT + F` and search for `’`, then replace with `'`.
> 
> Thank you for your continued hard work here pray

Hi Sam, 

I've reverted the changes in the team_communication.rst file. The changes were unintentional. I've fixed the file and image that was changed. In addition, I have removed the image from the Getting Started Folder. 

Thanks, 
Tim

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,87 +1,95 @@
-| Messages can then be *Marked as Todo*, *Replied*, or *Marked as Read*.
+By default, the :guilabel:`Notification` field is set as :guilabel:`Handle by Emails`. With this
+setting enabled, a notification email will be sent by Odoo every time a message is sent, a note is
+sent with an '@' mention, or a notification is sent for a record that the user follows.
```
I suppose this would be user input, so its probably best to format it as such
```suggestion
sent with an `@` mention, or a notification is sent for a record that the user follows.
```

samueljlieber commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,87 +1,95 @@
    :align: center
    :height: 400
-   :alt: View of discuss’s panel emphasizing the titles channels and direct messages in Odoo Discuss
+   :alt: View of Discuss's panel emphasizing the titles channels and direct messages in Odoo Discuss
```
Add a missing period
```suggestion
   :alt: View of Discuss's panel emphasizing the titles channels and direct messages in Odoo
         Discuss.
```

samueljlieber reviewed (approved):
Hi @tiku-odoo this PR is looking great!! Nice job reverting your edits, looks like `get_started` is the only file changed now 🙂 I have just two small improvements, please see below and make them before tagging for final review. Im approving now!

tiku-odoo commented:
@StraubCreative  This document is ready for your review when you have a moment. Thanks for your help on this review. 

Tim :+1:

tiku-odoo commented:
@meng-odoo This doc is ready for your review when you have a moment. 

Thanks. Tim :)

tiku-odoo commented:
CC @meng-odoo This doc is ready for your review :+1:

tiku-odoo commented:
@odoo/us-doc-review 

This PR is ready for your review! Thank you :)

tiku-odoo commented:
@odoo/us-doc-review 

Bumping this document to the top... needs a final review.

StraubCreative commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,87 +1,96 @@
-  they are shown in the *Inbox*.
-| Messages can then be *Marked as Todo*, *Replied*, or *Marked as Read*.
+By default, the :guilabel:`Notification` field is set as :guilabel:`Handle by Emails`. With this
+setting enabled, a notification email will be sent by Odoo every time a message is sent, a note is
+sent with an `@` mention, or a notification is sent for a record that the user follows.
```
be more specific
what kind of message, what kind of note, what is an example of a record, etc.

for people who know the software we can more or less piece this together but for people who know nothing about it and want to understand the mechanics of these notifications, fully cover the triggers and instances of how and when they fire.

StraubCreative reviewed (changes requested):
Hi @tiku-odoo 

I'd like to see the content here expanded more deeply and with more specific language.

The near 1:1 replacements of 2nd-person and small wording changes don't seem to be adding value to the doc; the new version is only about 10 lines longer than the previous one which suggests the content wasn't added to or expanded in a meaningful way. 

As well it doesn't appear we urgently corrected a mistake that needed a merge right away.

If you'd like, we can merge these changes in right away just to quickly update the language, and then do another PR expanding the content out more?

Lmk what you'd like to do :)

tiku-odoo commented:
@StraubCreative 
I'd like to push these changes to get the vital info updated and address the rest of the doc at a later time.

I've made the requested changes, all good to go! 

Thanks,
Tim

StraubCreative commented on content/applications/productivity/discuss/overview/get_started.rst:
```diff
@@ -1,87 +1,96 @@
+By default, the :guilabel:`Notification` field is set as :guilabel:`Handle by Emails`. With this
+setting enabled, a notification email will be sent by Odoo every time a message is sent from the
+chatter, a note is sent with an `@` mention (from chatter), or a notification is sent for a record
+that the user follows. Something that triggers a notification is changing of the stage (if an
+email is configured to be sent, for example if the task is set to :guilabel:`Done`).
```
```suggestion
that the user follows. Something that triggers a notification is changing of the stage (if an email
is configured to be sent, for example if the task is set to :guilabel:`Done`).
```

StraubCreative commented:
@robodoo r+
